Intersect (Core)/Threading/Threaded`1.cs

```diff
@@ -1,33 +1,32 @@
-﻿using System;
-
-namespace Intersect.Threading
+﻿namespace Intersect.Threading
 {
-
-    /// <inheritdoc />
+    /// <summary>
+    /// Represents a generic base class for creating and managing new threads.
+    /// </summary>
+    /// <typeparam name="TArgument">Type of argument that will be passed to a new thread.</typeparam>
```

Comment:
This line can stay, but with different wording. It should say
> The type of the startup argument that is expected to be passed into <cref to Thread.Start>.

Also please change TArgument to TStartupArgument.

(I may have gotten the cref wrong, but it should be pointing to `ThreadStart(params object[] args)`).